src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -36,6 +36,7 @@ missing.switch.default=switch without \"default\" clause.
 modified.control.variable=Control variable ''{0}'' is modified.
 multiple.statements.line=Only one statement per line allowed.
 multiple.string.literal=The String {0} appears {1} times in the file.
+multiple.string.literal.text.block=The String \"{0}\" appears {1} times in the file.
```

Comment:
Looks like this one is different than the others: `\"` vs. `\\"`, is that intentional?

Reply:
Removed.